src/petals/server/memory_cache.py

```diff
@@ -60,11 +61,14 @@ def handle_counter(self, value: int):
         self._handle_counter.value = value
 
     @contextlib.asynccontextmanager
-    async def allocate_cache(self, *descriptors: TensorDescriptor) -> AsyncContextManager[Sequence[Handle]]:
+    async def allocate_cache(
+        self, *descriptors: TensorDescriptor, timeout: Optional[float] = None
```

Comment:
```suggestion
        self, *descriptors: TensorDescriptor, timeout: float
```

Let's assume `timeout` to be known here, otherwise we'll have defaults in two places.

Reply:
Here, timeout=None is not a default to be replaced, it literally means no timeout